### PR TITLE
[IMP] mail: introduce CallView/tileWidth

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2871869

### CODE REVIEW

File: addons/mail/static/src/components/call_view/call_view.js [hunk 1]
```diff
@@ -109,7 +108,7 @@ export class CallView extends Component {
             tileCount: this.tileContainerRef.el.children.length,
         });
 
-        this.state.tileWidth = tileWidth;
+        this.callView._update({ tileWidth });
```
Comment (phenix-factory) [inline addons/mail/static/src/components/call_view/call_view.js:111]: Why do you use the private update here?

Comment (alexkuhn) [reply addons/mail/static/src/components/call_view/call_view.js:111]: Typo, will fix it.
IIRC, I was fighting with the autocompletion feature of IDE, and it still let me type the wrong things 🥲

Comment (alexkuhn) [reply addons/mail/static/src/components/call_view/call_view.js]: Changes done

**Review by phenix-factory — APPROVED** [review]
LGTM

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/93330

**Pull request closed without merging**